core/src/main/scala/kafka/server/Kip500Controller.scala

```diff
@@ -151,7 +151,7 @@ class Kip500Controller(val config: KafkaConfig,
         kafkaMetricsReporters ++ metrics.reporters.asScala)
 
       logManager = new LocalLogManager(new LogContext(),
-        config.controllerId, config.metadataLogDir, "", 10)
+        config.controllerId, config.metadataLogDir, "")
```

Comment:
compile error?

Reply:
Ah, ok, I see, you eliminated it

Reply:
It should be fixed. It's compiling w/ Scala 2.12 and 2.13